tests/integration/conftest.py

```diff
@@ -14,6 +21,12 @@
 from tests.integration.integration_steps.time_series_generation import generate_timeseries_set
 
 load_dotenv()
+RESOURCES = Path(__file__).parent / "resources"
+SPACE_ID = "IntegrationTestSpace"
```

Comment:
Already defined in config file, any way to reuse?

Reply:
Added a fixture to load the yaml config into an object.  Would like to hear from the rest of the team if this is a good approach